odoo/addons/base/views/res_users_identitycheck_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="identity_check_wizard" model="ir.ui.view">
```

Comment:
Guidelines: should be 'res_users_identitycheck_view_form', as it is a form view on a model... it helps understanding the content of a record by looking at its id (otherwise it might seem clear for you, but probably not for other people). See Odoo development guidelines for more details about naming convention :) .

Reply:
This entire record is something I've moved from res_users_views.xml to res_users_identitycheck_views.xml. Changing this external id would require a migration process as it is already used. It would probably be better to keep it that way.